infoschema/infoschema.go

```diff
@@ -59,6 +59,8 @@ type InfoSchema interface {
 	RuleBundles() []*placement.Bundle
 	// AllPlacementPolicies returns all placement policies
 	AllPlacementPolicies() []*model.PolicyInfo
+	// DismissPolicyDependencies return whether it's ok to drop/modify a policy.
+	DismissPolicyDependencies(name string) []int64
```

Comment:
The name and comments of `DismissPolicyDependencies` is a little weird to me.

`return whether it's ok to drop/modify a policy.` means it should return a bool, but it returns []int64. Name `DismissPolicyDependencies` looks more like an action, why not use GetXXXIDs instead?